app/src/main/java/com/twilio/video/app/ui/room/RoomManager.kt

```diff
@@ -0,0 +1,35 @@
+package com.twilio.video.app.ui.room
+
+import com.twilio.video.RemoteParticipant
+import com.twilio.video.Room
+import com.twilio.video.TwilioException
+
+class RoomManager : Room.Listener {
```

Comment:
Is this dead code?

Reply:
No it's used in future commits to abstract out room management. I just didn't do a good job separating this class from the UI test commits